diagnostic.md

```diff
@@ -5,74 +5,74 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```bash
-// your response here
+To store, manage and protect data and to allow communication between users.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```bash
-// your response here
+The controller layer
 ```
```

Comment:
The controller layer is use by the controller layer?

The model communicates with the model?

You mixed these two up ⇑⇓